public/components/settings/api/add-api.js

```diff
@@ -150,7 +151,8 @@ export class AddApi extends Component {
           <EuiCode>{'<api_port>'}</EuiCode> is the port,{' '}
           <EuiCode>{'<api_username>'}</EuiCode> and{' '}
           <EuiCode>{'<api_password>'}</EuiCode> are the credentials to
-          authenticate.
+          authenticate,{' '}
+          <EuiCode>{'<run_as>'}</EuiCode> defines if the app user permissions depends on the authentication context (<EuiCode>{'true'}</EuiCode> / <EuiCode>{'false'}</EuiCode>).
```

Comment:
spell correction
` defines if the app user's permissions...`

Reply:
👍